modules/ww-tritonnp/ww-tritonnp.wdl

```diff
@@ -0,0 +1,190 @@
+
+
+version 1.0
+
+# Import testdata module for automatic demo functionality
+import "https://raw.githubusercontent.com/caalo/wilds-wdl-library/refs/heads/ww-TritonNP/modules/ww-testdata/ww-testdata.wdl" as ww_testdata
+
+
+#### WORKFLOW DEFINITION ####
+
+workflow tritonnp_example {
+  meta {
+    author: "Chris Lo"
+    email: "clo2@fredhutch.org"
+    description: "TritonNP module."
+    url: "https://github.com/caalo/TritonNP"
+    outputs: {
+        final_composite: "Aggregrated outputs from all files",
+        fm_files: "Array of outputs from individal samples"
+    }
+  }
+
+  # Auto-download test data for testing purposes
+  call ww_testdata.download_tritonnp_data as download_demo_data { }
+
+  # Sample configuration
+  Array[String] sample_names = ["NA12878"]
+  Array[File] bam_paths = [download_demo_data.bam]
+  Array[File] bam_index_paths = [download_demo_data.bam_index]
+  Array[File] bias_paths = [download_demo_data.bias]
+  
+  # Global configuration parameters
+  String results_dir = "results"
+  File annotation = download_demo_data.annotation
+  File reference_genome = download_demo_data.reference
+  File reference_genome_index = download_demo_data.reference_index
+  File plot_list = download_demo_data.plot_list
+  Int map_quality = 20
+  String size_range = "15 500"
+  Int triton_main_ncpus = 4
```

Comment:
Since these are only used once I would just set the `triton_main` input parameters to these directly instead of defining them here first.